docs/source/index.rst

```diff
@@ -42,23 +42,23 @@ Substra has three user interfaces:
 Installation
 ^^^^^^^^^^^^
 
-Install substra and substrafl with the following command:
+**Client side**: Install Substra and SubstraFL python libraries with the following command: ``pip install substrafl``. Substra python library is a dependency of SubstraFL, so it will be automatically installed. More information on the installation :ref:`can be found here <substrafl_doc/substrafl_overview:Installation>`.
 
-.. code-block:: console
+**Server side**: There are two options to deploy the server side of Substra (backend, frontend and orchestrator):
```

Comment:
Perhaps mention that Substra can be used in subprocess or docker mode without a server?

Reply:
yes good point, I will add that